### PR TITLE
add cis sitemaps index

### PR DESCRIPTION
**Pull request by 3ch023**
- add sitemaps index config for CIS_RU and CIS_EN
- add missing cards index definitions for a number of geos

Resolves: https://jira.corp.adobe.com/browse/MWPW-140666 

**Test URLs:**
N/A

### CODE REVIEW

File: helix-sitemap.yaml [hunk 1]
```diff
@@ -565,3 +565,33 @@ sitemaps:
...
+        hreflang:
+          - ru-AM
+          - ru-AZ
+          - ru-BY
+          - ru-KZ
+          - ru-KG
+          - ru-MD
+          - ru-RU
+          - ru-TJ
+          - ru-UZ
```
Comment (Ruchika4) [inline helix-sitemap.yaml:597]: Please update the hreflang as shared by Sharon
hreflang=“ru-by” << cis_ru
hreflang=“en-by” << cis_en

Screen Shot 2024-03-05 at 9 11 55 PM

Comment (Ruchika4) [reply helix-sitemap.yaml:597]: Also for sitemap for new geos we'll need to update helix-query.yaml too 
And make sure query-index.xlsx is set, previewed and published.

As these are new geos once the PR is merged sitemaps will need to be generated using api and then those sitemap links need to be added to index file here during go live https://github.com/adobecom/cc/blob/stage/creativecloud/sitemap-index.xml

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

File: helix-sitemap.yaml (hunk 1 above)
Comment (3ch023) [reply helix-sitemap.yaml]: thx @Ruchika4 I updated the PR and also created index files for cis_ru and cis_en:

[cis_en](https://adobe.sharepoint.com/sites/adobecom/CC/Forms/AllItems.aspx?newTargetListUrl=%2Fsites%2Fadobecom%2FCC&viewpath=%2Fsites%2Fadobecom%2FCC%2FForms%2FAllItems%2Easpx&id=%2Fsites%2Fadobecom%2FCC%2Fwww%2Fcis%5Fen%2Fcc%2Dshared%2Fassets&viewid=820c5cd2%2Dfd43%2D4244%2D92c4%2Dbe6ed86f524e)

**Review by aishwaryamathuria — APPROVED** [review]

**Review by Ruchika4 — APPROVED** [review]

**Comment by Ruchika4** [comment]
Hi @3ch023 , this looks good. Will you be raising a separate PR to add these 2 new geos to sitemap index file https://github.com/adobecom/cc/blob/stage/creativecloud/sitemap-index.xml so that they show up in robots.txt?

**Comment by 3ch023** [comment]
> https://github.com/adobecom/cc/blob/stage/creativecloud/sitemap-index.xml

I wasn't sure if we could add them before doing reindex and publishing the files

**Pull request merged**